File: 7KYU/Fizz Buzz/solution.cs
using System;
using System.Collections.Generic;
using System.Linq;
public class FizzBuzz
{
	public static string[] GetFizzBuzzArray(int n)
	{
		if(n <= 0) throw new ArgumentOutOfRangeException();
		return Enumerable.Range(1, n)
						 .Select(Fizzbuzz)
						 .ToArray();
	}

	public static string Fizzbuzz(int n)
	{
		var result = "";
		if(n % 3 == 0)
			result += "Fizz";
		if(n % 5 == 0)
			result += "Buzz";
		return string.IsNullOrEmpty(result) ? n.ToString() : result;
	}
}

File: 7KYU/Fizz Buzz/solution.py
def fizz_buzz_it(n):
    if n % 3 == 0 and n % 5 == 0:
        return 'FizzBuzz'
    elif n % 3 == 0:
        return 'Fizz'
    elif n % 5 == 0:
        return 'Buzz'
    return n
        
def fizzbuzz(upper):
    return [fizz_buzz_it(i) for i in range(1, upper+1)]
